### ISSUE TITLE
do you know when hdf5 1.10 is anticipating release?

### ISSUE THREAD

**Issue by jreback** [h5py/h5py#670]
looking at this: http://docs.h5py.org/en/latest/swmr.html

OT:
the docs say h5py 2.5.0 supports this (with HDF5-1.9.233), but only builds with current master of ``h5py``

**Comment by tacaswell** [comment]
Last I heard they have pinned down the API, but still might make changes in the bit layout of the files.

https://hdfgroup.org/wp/2015/05/whats-coming-in-the-hdf5-1-10-0-release/ claims march 2016 with an alpha1 out 5 days ago.

**Comment by epourmal** [comment]
All,

We are planning to release HDF5 1.10.0 around March 31. It will have VDS and SWMR features that are available in the alpha releases now. I know that there were pull requests for both SWMR and VDS. It would be great if the new release of h5py has those features and is available around the time of the HDF5 1.10.0 release.

We are trying to keep HDF5 trunk stable while integrating and releasing new features. Currently only VDS is available in HDF5 trunk. Integrated SWMR and VDS (APIs are stable) are fully tested and reside in https://svn.hdfgroup.uiuc.edu/hdf5/branches/revise_chunks/ or in https://svn.hdfgroup.uiuc.edu/hdf5/branches/hdf5_1_10_alpha1/ (next will be alpha2, etc). We are in the process of integrating those features with the parallel enhancements and that is why the HDF5 file format is not stable in alphas.

If you are working on integrating SWMR and VDS with h5py, please use our alpha branches and not old 1.9.233 version as was mentioned before.

And let me know if you have any questions about the new features.

Thank you!

Elena
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Elena Pourmal  The HDF Group  http://hdfgroup.org
1800 So. Oak St., Suite 203, Champaign IL 61820
217.531.6112
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




On Jan 25, 2016, at 9:09 PM, Thomas A Caswell <notifications@github.com<mailto:notifications@github.com>> wrote:


Last I heard they have pinned down the API, but still might make changes in the bit layout of the files.

https://hdfgroup.org/wp/2015/05/whats-coming-in-the-hdf5-1-10-0-release/ claims march 2016 with an alpha1 out 5 days ago.
